extensions/schema-compare/src/schemaCompareMainWindow.ts

```diff
@@ -725,7 +725,7 @@ export class SchemaCompareMainWindow {
 			TelemetryReporter.sendActionEvent(TelemetryViews.SchemaCompareMainWindow, 'SchemaCompareOptionsOpened');
 			// create fresh every time
 			this.schemaCompareOptionDialog = new SchemaCompareOptionsDialog(this.deploymentOptions, this);
-			await this.schemaCompareOptionDialog.openDialog();
+			this.schemaCompareOptionDialog.openDialog();
```

Comment:
Just an FYI, I have updated the openDialog method and it is async now- so will be needing an await here. But may be for this PR, go ahead with these changes and I will update it in a separate PR :).